docs/src/main/paradox/java/http/routing-dsl/directives/basic-directives/extract.md

```diff
@@ -1,5 +1,13 @@
 # extract
 
+@@@ div { .group-scala }
```

Comment:
Why put Scala signatures in the Java version?

Reply:
Basically my strategy is to make the difference between the Java and Scala versions as small as possible until they are the same and the Java version can be replaced by a symlink